app/src/helpers/url.js

```diff
@@ -2,7 +2,7 @@ export function getURLParams(window) {
   const queries = window.location.search.slice(1).split(`&`)
   return queries.reduce((config, current) => {
     const [name, value] = current.split(`=`)
-    if ([`stargate`, `rpc`].includes(name)) {
+    if ([`stargate`, `rpc`, `dev`].includes(name)) {
```

Comment:
`dev` or `devMode`?

Reply:
I could go with either.

Reply:
was also thinking lately that perhaps we should change this from `dev` to `insecure` as i don't like conflating "developer mode / feature flags" with the sign in / seed changes. but this can be discussed as a separate issue.

Reply:
> I could go with either.

let's use `devMode` then - as this matches the property in the module.